workers/unity/Assets/Playground/Config/CubeTemplate.cs

```diff
@@ -29,13 +22,14 @@ public static Entity CreateCubeEntityTemplate(Coordinates coords)
                 .AddMetadata(entityType, WorkerUtils.UnityGameLogic)
                 .SetPersistence(true)
                 .SetReadAcl(WorkerUtils.AllWorkerAttributes)
-                .AddComponent(transform, WorkerUtils.UnityGameLogic)
                 .AddComponent(cubeColor, WorkerUtils.UnityGameLogic)
                 .AddComponent(cubeTargetVelocity, WorkerUtils.UnityGameLogic)
-                .AddComponent(launchable, WorkerUtils.UnityGameLogic)
-                .Build();
+                .AddComponent(launchable, WorkerUtils.UnityGameLogic);
+
+            TransformSynchronizationEntityBuilderHelper.AddComponents(entity, WorkerUtils.UnityGameLogic,
```

Comment:
This class name feels... long.

/me braces for big discussion

How about `TransformSynchronizationModule.AddEntityComponents`?

Reply:
Or extension methods? (😅)

Reply:
I like this^

we could probably then merge {module}SystemHelper and {module}EntityBuilderHelper classes into the same class